multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/util/DynamicParameterUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.cloudfoundry.multiapps.mta.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+public final class DynamicParameterUtil {
+
+    public static final String PATTERN_FOR_DYNAMIC_PARAMETERS = "'{'ds/{0}/{1}'}'";
+    public static final String REGEX_PATTERN_FOR_DYNAMIC_PARAMETERS = "^\\{ds[A-Za-z0-9_\\-\\.\\/]+\\}$";
+
+    private DynamicParameterUtil() {
+    }
+
+    public static String getRelationshipName(String dynamicParameter) {
+        return dynamicParameter.substring(StringUtils.ordinalIndexOf(dynamicParameter, "/", 1) + 1,
+                                          StringUtils.ordinalIndexOf(dynamicParameter, "/", 2));
+    }
+
+    public static String getParameterName(String dynamicParameter) {
+        return dynamicParameter.substring(StringUtils.ordinalIndexOf(dynamicParameter, "/", 2) + 1, dynamicParameter.lastIndexOf("}"));
```

Comment:
Same here.
This _`ordinalIndexOf`_ seems overly complicated of a name, when we really just need the first and/or last index of the slash